content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -11,6 +11,10 @@ weight: 65
 
 {{< feature-state feature_gate_name="DynamicResourceAllocation" >}}
 
+Dynamic Resource Allocation with ResourceClaim device status:
+
+{{< feature-state feature_gate_name="DRAResourceClaimDeviceStatus" >}}
```

Comment:
In https://github.com/kubernetes/website/pull/48330 I decided to document the "admin access" feature under separate sections. IMHO that made the rendering of the feature gate state a bit more readable.

Reply:
Done, thank you for the comment and sorry for the delay.